### DISCUSSION TITLE
How to add new state and district and modify existing  location data

### DISCUSSION THREAD

**Discussion by jayrajk112** [Sunbird-Lern/Community#175] [Issues]
Hi team,

We need to add some new locations (district and block), So please suggest how to be added the same in the system.
We need also modify the existing location data in Diksha platform.
Can you let us know earlier how to be this activity has been performed?

Thanks,
Jayraj Kakde

**Comment by Jayaprakash8887** [comment]
@jayrajk112 Kindly go through https://lern.sunbird.org/use/developer-guide/user-and-org-service/apis/location-management

Related to DIKSHA data, you would have received necessary scripts and documentation from Ex-DIKSHA Implementation team. Kindly check for the same.

**Reply by jayrajk112** [reply]
we have connected with diksha implementation team, but they don't have any script. can you please help us how to be get this script and documentation.

**Comment by jayrajk112** [comment]
we have connected with diksha implementation team, but they don't have any script. can you please help us how to be get this script and documentation.

**Reply by Jayaprakash8887** [reply]
@jayrajk112 As mentioned earlier modification of data can be achieved via APIs. Details of APIs has been shared above. 
For onboarding new locations to DIKSHA, please refer to 
KT : https://drive.google.com/file/d/1gyibY_1lK-0C_g9PgN1yNqLbaZsS-79X/view?usp=share_link
Script: https://github.com/DIKSHA-NCTE/Ops-Tool/tree/main/Standalone-tools/location_school_onboard
Process doc: https://drive.google.com/file/d/1z4WGTK3obh8bC43p8pWWisp40mMEEaUK/view

**Reply by jayrajk112** [reply]
@Jayaprakash8887  Thanks for your response
